etc/leapp/files/repomap.json

```diff
@@ -105,6 +105,13 @@
                     "target": [
                         "rhel8-rhui-google-compute-engine-leapp"
                     ]
+                },
+                {                   
+                    "source": "rhui-custom-rhui_client_at_alibaba",
```

Comment:
Not sure change to repomap.json file belongs here :(
Yes, we started shipping leapp data files with the package, but repomapping is automatically generated and changing it here will last only till the next version is shipped.
I believe we have to change the data files that are used to generate the `repomap.json`, not the file itself. The problem is - they are not in this repo, so I believe this should be done by us -> the repomap.json should be regenerated -> the change will be shipped in a different commit.
@pirat89 @mm0ran @mmacura311 @Rezney what's your opinion on this?

Reply:
@fernflower it's not supposed to be merged yet - it's waiting for @Rezney  when he returns from the vacation. yes, once it's decided this is going to be merged, the new data must be generated - for that, we will need to get "source of truth" for alibaba repositories.

@jinkkkang can you please provide repofiles from alibaba machine which we could use for our internal script? as the repomap data is generated, without updating our internals repositories would get removed if we do not update the script. regarding additional internal we use in the script, we cannot make it public, only the data

Reply:
@fernflower repomap file has been updated and some other changes in the code applied. see my 2nd commit.